stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -63,11 +64,14 @@
     private static final String LOGGING_ENDPOINT = "https://m.stripe.com/4";
 
     private static final String CHARSET = "UTF-8";
+    private static final String CUSTOMERS = "customers";
     private static final String TOKENS = "tokens";
     private static final String SOURCES = "sources";
     private static final String DNS_CACHE_TTL_PROPERTY_NAME = "networkaddress.cache.ttl";
     private static final SSLSocketFactory SSL_SOCKET_FACTORY = new StripeSSLSocketFactory();
 
+    static final String API_VERSION = "2017-06-05";
```

Comment:
We can, but that probably shouldn't be in this PR. Also, don't we want to hit the API that the user is expecting to hit, since that's what they'll be hitting with their own backend code?